schema/deploy/computed_columns/form_change_eligible_expenses_to_date.sql

```diff
@@ -0,0 +1,21 @@
+-- Deploy cif:computed_columns/form_change_eligible_expenses_to_date to pg
+-- requires: tables/form_change
+
+begin;
+
+create or replace function cif.form_change_eligible_expenses_to_date(parameter_fc cif.form_change)
```

Comment:
this function will return the same value for all milestones, should we only sum the milestones that have been submitted before / including this milestone?

Reply:
These functions are no longer calculating these values for individual milestones. They are for the budgets, payments and expenses page. So they're meant to be a total sum of all milestones.